src/v2/Apps/Consign/Components/InDemandNow/ConsignTopArtists.tsx

```diff
@@ -85,11 +90,48 @@ const TopArtists: React.FC<ConsignTopArtistsQuery["response"]> = props => {
     return null
   }
 
+  // Iterate over microfunnel artworks and a) group into "chunks" of four for
+  // displaying in rows; b) average each artwork realized price; c) shuffle the
+  // display so that its a bit less static.
   const recentlySoldArtworks = chunk(
     shuffle(
-      microfunnelItems.map(
-        artwork => artwork?.artworksConnection?.edges?.[0]?.node
-      )
+      microfunnelItems.map(artwork => {
+        if (artwork?.artworksConnection?.edges?.length === 0) {
+          return null
+        }
+
+        const { edges } = artwork.artworksConnection
+
+        // Get realized price average for artworks
+        const realizedPriceAverage = edges.reduce(
+          (total, artwork, index, artworks) => {
+            // Done adding up realized prices, now average
+            if (index === artworks.length - 1) {
+              const averageInDollars = formatCentsToDollars(
+                (total / artworks.length) * 100
+              )
+              return averageInDollars
+            }
+
+            // Value comes back from MP formatted; strip dollars and commas
+            const realizedPrice = artwork.node.realizedPrice
+              .replace("$", "")
```

Comment:
Are we guaranteed dollar amounts? E.g. could we ever end up with GBP/EUR symbols?

Reply:
Similar with the `,` below, wondering if we ever format numbers with `.` instead

Reply:
> Are we guaranteed dollar amounts? E.g. could we ever end up with GBP/EUR symbols?

Everything in the spreadsheet is in dollars, but if we were ever to update it to be truly dynamic it could be a concern.

Reply:
Ah okay, these are the curated works -- I'm cool with leaving as-is then.